src/equiv.js

```diff
@@ -59,6 +59,10 @@ QUnit.equiv = (function() {
 		return false;
 	}
 
+	function getRegExpFlags( regexp ) {
+		return regexp.flags || regexp.toString().match(/[gimuy]*$/)[0];
```

Comment:
@leobalter 
Thank you. When I ran `grunt` It have been success. So I wonder that why this case not found it in `jscs` task. I found cause. Current It normally doesn`t work `jscs task` because `grunt-jscs` version is low. If `jscs` version is high It is normally working.  I opened #917 for this case.
I have a question.  Can i change up version of `grunt-jscs`?  If it do task I going to change files.
I going to apply rules independently of #917.

@gibson042 
> I'd prefer "flags" in regexp ? regexp.flags : …
:+1: 
This pr that CI ran is success. How can i test more environments?